src/js/components/Data/stories/StarWars.js

```diff
@@ -37,9 +38,12 @@ export const StarWars = () => {
     abortRef.current = new AbortController();
     fetchData(view, abortRef.current.signal).then(({ count, results }) => {
       if (results) {
+        // The API doesn't provide a non-filtered total, so we rely on the
+        // first call having no filtering telling us the total.
+        setTotal(prevTotal => Math.max(prevTotal, count));
         setResult({
           data: results,
-          total: count,
+          filteredTotal: count,
```

Comment:
We need both `filteredTotal` and `total` in Data so that DataSummary has the content. Pagination needs filteredTotal to determine the number of pages.